sw/device/rom_exts/rom_ext_start.S

```diff
@@ -117,12 +120,117 @@ _rom_ext_interrupt_vector:
   .option norelax
 
 /**
- * Entry point after reset. This symbol is jumped to from the handler
- * for IRQ 32.
+ * Entry point.
  *
- * At the moment, this should just fault, until we have a CRT we can use.
+ * This symbol is jumped to from `_rom_ext_interrupt_vector + 32`, and
+ * is part of a two jump Mask ROM execution handover to ROM_EXT (`mask_rom_boot`
+ * -> `_rom_ext_interrupt_vector + 32` -> `_rom_ext_start_boot`).
```

Comment:
Nit: I think `_rom_ext_interrupt_vector + 32` is misleading because that makes it sound like it's 32 bytes later, can you clarify to `_rom_ext_interrupt_vector + (32 * 4) bytes` or just `_rom_ext_interrupt_vecotr + 128 bytes`.

Reply:
Good point, will do.

Reply:
Done.